certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -1,965 +1,1070 @@
+                    }
+                },
+                "x-ms-visibility": "internal"
+            }
         },
-        "EntityNameForAPI": {
-          "description": "Entity name that is used in CSS API",
-          "type": "string"
-        }
-      }
-    },
-    "CSSAPICustomMessage": {
-      "required": [
-        "LogId",
-        "Message"
-      ],
-      "type": "object",
-      "properties": {
-        "LogId": {
-          "type": "string"
+        "/CSSAPI/v2/PowerAutomate/PollTriggers/{entityName}/NewRecords": {
+            "post": {
+                "tags": [
+                    "Poll Triggers"
+                ],
+                "summary": "A scheduled trigger that query created (new) records on a given interval",
+                "description": "Query all newly created records by a date & time field compared to current date & time (greater than or equal to) on a specific interval.",
+                "operationId": "CSSAPINewRecordsPollTrigger",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "x-ms-trigger": "batch",
+                "x-ms-trigger-metadata": {
+                    "mode": "polling",
+                    "kind": "query"
+                },
+                "parameters": [
+                    {
+                        "name": "entityName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "x-ms-summary": "Entity Name",
+                        "x-ms-url-encoding": "single",
+                        "description": "Table name to pull new record(s) from.",
+                        "x-ms-dynamic-values": {
+                            "operationId": "CSSAPIEntityListUpdate",
+                            "value-path": "EntityNameForAPI",
+                            "value-title": "EntityNameDisplay"
+                        }
+                    },
+                    {
+                        "name": "since",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "format": "date-time",
+                        "x-ms-visibility": "internal"
+                    },
+                    {
+                        "name": "query",
+                        "in": "body",
+                        "description": "Query Model",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/APIQueryRequest"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/APIQueryResponse"
```

Comment:
@nizarhandal The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please confirm this change?